README.md

```diff
@@ -4,11 +4,28 @@ Gradle plugin that generates Jacoco reports from a Gradle Project. Android Appli
 
 ### Android project
 
+*JVM Unit-Tests*
 - Task `jacocoTestReport<Flavor><BuildType>`
   - Executes the `test<Flavor><BuildType>UnitTest` task before
   - Gets executed when the `check` task is executed
   - Generated Jacoco reports can be found under `build/reports/jacoco/<Flavor>/<BuildType>`.
 
+*Instrumentation tests*
+- Task `combinedTestReport<Flavor><BuildType>`
+  - Executes the `test<Flavor><BuildType>UnitTest` and `create<Flavor><BuildType>CoverageReports` tasks before (JVM and instrumentation tests)
```

Comment:
also capital `Instrumentation`?

Reply:
The documentation speaks of `instrumented tests` (lowercase). I'll use that.